examples/dreambooth/train_dreambooth.py

```diff
@@ -241,6 +241,7 @@ def parse_args(input_args=None):
         ),
     )
     parser.add_argument("--local_rank", type=int, default=-1, help="For distributed training: local_rank")
+    parser.add_argument("--use_xformers", action="store_true", help="Whether or not to use xformers.")
```

Comment:
could call this arg `enable_xformers_memory_efficient_attention` to align with the method name ?